src/java.base/share/classes/java/nio/file/Path.java

```diff
@@ -845,25 +845,25 @@ WatchKey register(WatchService watcher,
      *
      * <p> An invocation of this method behaves in exactly the same way as the
      * invocation
-     * <pre>
-     *     watchable.{@link #register(WatchService,WatchEvent.Kind[],WatchEvent.Modifier[]) register}(watcher, events, new WatchEvent.Modifier[0]);
-     * </pre>
+     * {@snippet lang=java :
+     *     register(watcher, events, new WatchEvent.Modifier[0]);
```

Comment:
It looks like the link has been lost, try changing it to:
```
// @link substring="register" target="Watchable#register" :
register(watcher, events, new WatchEvent.Modifier[0]);
```

Reply:
Thanks for the tip. Fixed in https://github.com/openjdk/jdk/pull/13870/commits/acd62d5ccadd0667b2b08aae9de531adbfdcc8b1.